openslides/core/static/js/core/site.js

```diff
@@ -1180,6 +1189,18 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Privacy Policy Controller
+.controller('PrivacyPolicyCtrl', [
+    '$scope',
+    '$http',
+    function ($scope, $http) {
+        $http.get('/core/version/').then(function (success) {
+            $scope.core_version = success.data.openslides_version;
+            $scope.core_license = success.data.openslides_license;
```

Comment:
The `cure_url` is missing here.

Reply:
'''cure_url'''? Can't find that in the code. Can you elaborate?

Reply:
Sorry `core_url`. See comment below. Its used in the template but nowhere defined.